test/parallel/test-async-hooks-async-await.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
```

Comment:
This needs a `require('../common')` as the first require

Reply:
Also, it would be helpful to have a comment in here that explains what this is testing for.